.github/workflows/release-packages.yaml

```diff
@@ -149,35 +149,35 @@ jobs:
           SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
           SLACK_MESSAGE: "${{ job.status == 'success' && 'Ubuntu 20.04 package built and uploaded successfully' || 'Ubuntu 20.04 package build failed' }}"
 
-  homebrew-pr:
-    runs-on: macos-11
-    steps:
-      - name: Get release tag name
-        # The GITHUB_REF we get has refs/tags/ in front of the tag name so we
-        # strip that here
-        run: echo "RELEASE_TAG=${GITHUB_REF/refs\/tags\/}" >> $GITHUB_ENV
-      - name: Configure git user name and email
-        uses: Homebrew/actions/git-user-config@07da0794847043a11761f14c97cc682577c74d5d
-        with:
-          username: db-ci-cprover
-      - name: Create homebrew PR
-        run: |
-          brew update-reset
-          brew bump-formula-pr --tag "$RELEASE_TAG" --revision "$GITHUB_SHA" cbmc
-        env:
-          HOMEBREW_GITHUB_API_TOKEN: ${{ secrets.DB_CI_CPROVER_ACCESS_TOKEN }}
-      - name: Checkout CBMC project source code to obtain access to scripts
-        if: always()
-        uses: actions/checkout@v3
-      - name: Slack notification of CI status
-        if: success() || failure()
-        env:
-          SLACK_CHANNEL: team_open_source
-          SLACK_COLOR: ${{ job.status }}
-          SLACK_USERNAME: Github Actions CI bot
-          SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
-          SLACK_MESSAGE: "${{ job.status == 'success' && 'Homebrew PR submitted successfully' || 'Homebrew PR failed' }}"
-        run: go run scripts/slack_notification_action.go
+  # homebrew-pr:
```

Comment:
I will consider this for a next PR, but I want to move this one forward to unblock us.

In the meantime, I'm hoping that someone coming across this change would see the `blame` long which would lead him to the commit https://github.com/diffblue/cbmc/pull/7987/commits/2d14e9996f197f4373d8438a300713a66307ebfe and its message that explains the situation.